src/test/java/com/wikia/webdriver/testcases/mercurytests/InteractiveMapsTests.java

```diff
@@ -59,77 +79,68 @@ public void MercuryInteractiveMaps_003_ZoomByGesture() {
     maps.clickViewMapButton();
     maps.switchToMapFrame();
     Assertion.assertFalse(maps.isZoomInButtonEnabled(), "Zoom in button is enabled");
-    touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_OUT, WAIT_TIME);
+    PageObjectLogging.log("Zoom in button", "is disabled", true);
+    File beforeZooming = new Shooter().capturePage(driver);
+    maps.clickZoomOut();
+    base.waitMilliseconds(5000, "Wait after zoom out");
+    File afterZooming = new Shooter().capturePage(driver);
+    Assertion.assertFalse(new ImageComparison().areFilesTheSame(beforeZooming, afterZooming),
+                          "Zoom out doesn't work");
+    PageObjectLogging.log("Zoom out by click", "works", true);
     Assertion.assertTrue(maps.isZoomInButtonEnabled(), "Zoom in button is disabled");
-    touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_IN, WAIT_TIME);
+    PageObjectLogging.log("Zoom in button", "is enabled", true);
+    beforeZooming = new Shooter().capturePage(driver);
+    maps.clickZoomIn();
+    base.waitMilliseconds(5000, "Wait after zoom in");
+    afterZooming = new Shooter().capturePage(driver);
+    Assertion.assertFalse(new ImageComparison().areFilesTheSame(beforeZooming, afterZooming),
+                          "Zoom in doesn't work");
+    PageObjectLogging.log("Zoom in by click", "works", true);
     Assertion.assertFalse(maps.isZoomInButtonEnabled(), "Zoom in button is enabled");
+    PageObjectLogging.log("Zoom in button", "is disabled", true);
+    beforeZooming = new Shooter().capturePage(driver);
+    touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_OUT, 5000);
+    afterZooming = new Shooter().capturePage(driver);
+    Assertion.assertFalse(new ImageComparison().areFilesTheSame(beforeZooming, afterZooming),
+                          "Zoom out doesn't work");
+    PageObjectLogging.log("Zoom out by gesture", "works", true);
+    Assertion.assertTrue(maps.isZoomInButtonEnabled(), "Zoom in button is disabled");
+    PageObjectLogging.log("Zoom in button", "is enabled", true);
+    beforeZooming = new Shooter().capturePage(driver);
+    touchAction.zoomInOutPointXY(50, 50, 50, 100, PerformTouchAction.ZOOM_WAY_IN, 5000);
+    afterZooming = new Shooter().capturePage(driver);
+    Assertion.assertFalse(new ImageComparison().areFilesTheSame(beforeZooming, afterZooming),
```

Comment:
In some cases - if zoom is broken (Let's say instead of zooming, you will move a view by 1px), test will finish with positive result. Maybe consider using something like this to check if page is zoomed ```$(window).width()/screen.width```
Probably you should add some pixel ratio scaling for mobile devices, but it's worth trying , should be a lot faster then comparing screenshots